### DISCUSSION TITLE
Don't like move of send later status message to spaces toolbar

### DISCUSSION THREAD

**Discussion by jikamens** [Extended-Thunder/send-later#483] [General]
I literally never use the spaces toolbar and have it hidden. I do, however, have the status bar at the bottom of the TB window displayed all the time. So moving the send later status in the new beta release from the bottom bar to the spaces bar means that I either can't see it or have to show the spaces bar and let it take up space in my TB window unnecessarily just so I can see it.

**Comment by jph-sendlater** [comment]
I agree that it's an adjustment, and I don't like the current implementation where clicking on the icon takes you to the options page. Ideally it would take you to your drafts folder, or maybe to a custom page that lists all the scheduled messages (next time I get a dedicated weekend for working on it, that's my plan). For now I'll likely move back to the status bar widget before putting this version on ATN for that reason.

That said, I think in the long run it'll need to live in the spaces toolbar. My understanding is that there's no plan to support status bar elements via an official mailextensions API. I don't think full mailextension adoption will be possible any time soon, but I would expect that within the next year or so we'll have enough mailextension support to drop the remaining "experiment" code, and the status bar api can't be the one dangling exception.

Tagging @jobisoft because he might have more insight on this.

**Comment by jobisoft** [comment]
The spaces toolbar will become a core UI element, for example to switch between calendar and mails. IIRC, spaces will become grouped soon, so if you are in the mail space, you will not see tabs of other spaces. 

You still have the option to use a browser_action to add a button to the main toolbar, which can be used to display status information as well (via its icon and/or badge).

The status bar is indeed something the UX team does not want to expose to add-ons.

**Reply by jph-sendlater** [reply]
Using a browser_action is an interesting idea. I'll give that a go and see how it feels.

**Reply by jph-sendlater** [reply]
Yep, I just tried it and it looks way better than the spacesToolbar. Thanks for the suggestion!

**Reply by jobisoft** [reply]
Stand-alone message widows do not have a browser_action by default, but you can force it to show up there as well, by defining `default_windows` in its manifest entry:
https://webextension-api.thunderbird.net/en/stable/browserAction.html#manifest-file-properties

**Comment by jikamens** [comment]
This has been resolved, or at least it's as resolved as it's going to get. It is what it is. ;-)
